packages/grid/_modules_/grid/hooks/features/useColumnResize.tsx

```diff
@@ -1,204 +1,130 @@
 import * as React from 'react';
+import { useEventCallback, ownerDocument } from '@material-ui/core/utils';
```

Comment:
We don't use those in the whole grid, it would be great if we could follow the convention.

Reply:
We use them in https://github.com/mui-org/material-ui-x/blob/64f0af18882acf4741228f433a913502c90d3d2a/packages/grid/_modules_/grid/components/AutoSizer.tsx#L2

>  could follow the convention.

https://github.com/mui-org/material-ui-x/pull/406/files/39e4c68c140b24b9d97a9b8ff2b2e2809ce3a9cc#r503492471